docs/blog/2022-07-20-vex-silence-false-positives.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "VEX: Permanently Silence False Positives with Vulnerability EXchange"
+description: "VEX (Vulnerability EXchange) is a new standard developed by the NTIA to exchange information about which vulnerabilities are actually exploitable in a product."
+slug: vex-silence-false-positives
+date: 2022-05-25T01:37
+keywords: [vex, sbom, "false positive", vulnerability]
+tags: [malware, dependencies, vulnerability, sbom, vex]
+authors: [alex]
+---
+
+<!--
+  ~ Copyright by LunaSec (owned by Refinery Labs, Inc)
+  ~
+  ~ Licensed under the Creative Commons Attribution-ShareAlike 4.0 International
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License. You may obtain a copy of the License at
+  ~
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+
+## VEX: Permanently Silence False Positives with Vulnerability EXchange
+
+As we highlighted in our post on False Positives in Vulnerability Scanning, the output of vulnerability assessment tools like npm audit can be cumbersome because they lack context. VEX (Vulnerability EXchange) is a new standard developed by the NTIA to exchange information about which vulnerabilities are actually exploitable in a product.
+
+<!--truncate-->
+
+### The Problem: False Positives
+
+npm audit is a good tool for checking your javascript dependencies against known vulnerabilities. However, it commonly results in alerts for vulnerabilities that are inaccessible at runtime or otherwise irrelevant.[^1] This combined with the fact that npm audit is run after install by default can result in users ignoring the output because the Signal to Noise Ratio is not high enough.[^2]
+
+
+### A Solution: VEX
+
+The NTIA is developing a standard containing the minimum information for exchanging security advisories in a machine readable format called VEX.[^3] Besides being a clever bit of wordplay, VEX documents enable consumers to enhance the accuracy of their vulnerability assessment tools.
+
+> **vex**  /veks/ • _verb_
+> 
+> make (someone) feel annoyed, frustrated, or worried, especially with trivial matters.
+
+VEX documents contain a list of vulnerabilities and their status in a product. These statuses can be unknown, affected, or not affected. They can contain partial or full results of a vulnerability assessment. Multiple sources may issue VEX documents. VEX documents are usually associated with an SBOM, but can be distributed separately.
```

Comment:
How about a nice example like: Let's say you make X and your customers scan results are showing up with a vulnerable package.  You can distribute a vex that says you CHECKED if that was vulnerable and made sure it wasn't.  Ideally, a customers scanner could ingest that information.